Backpack/Map/Classes/BPKMapAnnotation.m

```diff
@@ -0,0 +1,60 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKMapAnnotation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation BPKMapAnnotation
+
+- (instancetype)init {
+    self = [super init];
+
+    if (self) {
+        [self setup];
+    }
+
+    return self;
+}
+
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate {
```

Comment:
Why aren't these inits documented in `BPKMapAnnotation.h`?

Reply:
Because they're just overriding the underlying initialisers.
Can document them if that's helpful?